MapboxMobileEvents/MMEAPIClient.m

```diff
@@ -75,6 +77,11 @@ - (void)postEvents:(NSArray *)events completionHandler:(nullable void (^)(NSErro
             if (completionHandler) {
                 error = error ?: statusError;
                 completionHandler(error);
+              
+                [self.metricsManager metricsFromEvents:events error:error];
```

Comment:
Something I didn't pick up before - it's not particularly clear from the method name, what's going on here?
As it stands is sounds like something is going to be returned from `metricsFrom...:error:` - what does it do?

Reply:
Perhaps it should be something like `updateMetricsWithEvents:error`?

Reply:
Good point -- should have caught that.